### DISCUSSION TITLE
How to disable CSS vendor-prefixes test snapshots?

### DISCUSSION THREAD

**Discussion by riteshoyo** [necolas/react-native-web#2182] [Q&A] [answered]
Working on code sharing b/w React.js based website and react-native based mobile application. So, started using this library for running react-native syntax on web.

Vendor prefixes are added automatically that is mentioned [here](https://necolas.github.io/react-native-web/docs/browser-compatibility/#css)). 

The issue that i am facing is: Vendor prefixes are generated by react-native-web but those doesn't exist in case of react-native. So, my snapshot tests fails when test cases are run for different platforms. Is it possible to disable this vendor prefixing in test environment?

**Comment by necolas** [comment]
No. And you either shouldn't be rendering down to DOM in cross-platform snapshots, or shouldn't be sharing snapshots across platforms.

**Reply by riteshoyo** [reply]
@necolas Thanks for the reply. I am using "preset": "react-native-web" in case of website due to which additional vendor prefixes are also part of the snapshot which doesn't work in case of react-native.

It seems that generating different snapshot is the only option I've now. Please suggest if there is another approach I can go with.

**Comment by necolas** [comment, marked as answer]
Duplicate of https://github.com/necolas/react-native-web/discussions/1967
